packages/web-app-admin-settings/src/components/Users/SideBar/EditPanel.vue

```diff
@@ -201,6 +201,12 @@ export default defineComponent({
     const isInputFieldReadOnly = (key: string) => {
       return capabilityStore.graphUsersReadOnlyAttributes.includes(key)
     }
+    const isLoginToggleHidden = computed(() => {
+      const list = (capabilityStore.graphUsersReadOnlyAttributes ?? []).map((s: string) =>
+        s.trim().toLowerCase()
+      )
+      return list.includes('user.accountenabled') || list.includes('accountenabled')
+    })
```

Comment:
You can simply use `capabilityStore.graphUsersEditLoginAllowedDisabled` once you implemented the computed in the store, which make this one obsolete.